src/ResourceManager/Batch/Commands.Batch/WorkItems/GetBatchWorkItemCommand.cs

```diff
@@ -0,0 +1,85 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Batch;
+using Microsoft.Azure.Commands.Batch.Models;
+using Microsoft.Azure.Commands.Batch.Properties;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Constants = Microsoft.Azure.Commands.Batch.Utils.Constants;
+
+namespace Microsoft.Azure.Commands.Batch
```

Comment:
Updated to using ExecuteCmdlet()

Regarding putting everything into a client method, how does that work when the two parameter sets have different output types (single object vs. collection object)?  I'd rather not have the method have output type object, so it seems like I need to do some parameter processing in the cmdlet to choose the correct client method.  Looking at Storage's Get-AzureStorageBlob for example, it seems like they're doing something similar with choosing the method to use based on parameter set.  Is there a different example I should be following?